### PR TITLE
Hw34

### PR DESCRIPTION
**Pull request by margarita-ignatieva**

### CODE REVIEW

**Review by KovalAlexey — APPROVED** [review]

**Review by dmytro-hryhoruk — COMMENTED** [review]
image
make the same with your constructors in User

File: src/main/java/com/internet/shop/model/User.java [hunk 1]
```diff
@@ -10,12 +10,16 @@
...
         this.password = password;
     }
 
+    public User(String name, String login, String password) {
+        this(login, password);
+        this.name = name;
+    }
+
     public User(Long userId, String name, String login, String password, Set<Role> roles) {
         this.userId = userId;
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/internet/shop/model/User.java:25]: and here?

Comment (margarita-ignatieva) [reply src/main/java/com/internet/shop/model/User.java:25]: done

Comment (dmytro-hryhoruk) [reply src/main/java/com/internet/shop/model/User.java:25]: you have constructor with three of these parameters(name, login, password)

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**